.github/workflows/label-releases.yml

```diff
@@ -3,10 +3,17 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
+
 jobs:
   label-release:
+    permissions:
+      pull-requests: write # required for adding labels to PRs
     if: ${{ startsWith(github.event.pull_request.title, 'release:') }}
     runs-on: ubuntu-latest
     steps:
       - run: echo this is a release PR
       - run: gh pr edit ${{ github.event.pull_request.number }} --add-label release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm curious if this is working currently without passing `GITHUB_TOKEN`?

Reply:
It likely did not, I disabled the workflow a while ago since it was unused and working with untrusted input with a `pull_request_target` trigger. I opened a PR to remove it: https://github.com/open-telemetry/opentelemetry-js/pull/5772 🙂